### ISSUE TITLE
User Feedback: Annotations

### ISSUE THREAD

**Issue by brotatotes** [Church-Life-Apps/Songs#103]
Idea: Some way to add annotations to songs that could include definitions, poetic analysis, verse references, etc. ex. In this song, song 62, verse 4: "own" is used not as possessive but as "admit or acknowledge that something is the case or that one feels a certain way." and "crown" is used as a verb, not a noun. This isn't obvious upon singing it.

**Comment by brotatotes** [comment]
I don't think this is feasible short term. Would require a back end to save such annotations as well as some sort of review/approval system to prevent malicious/bad annotations.

**Comment by brotatotes** [comment]
Also not sure if it's a good idea to allow people to add their own subjective annotations on songs

**Comment by 123asian** [comment]
Maybe there could just be something personal that people could have saved
locally?

**Comment by brotatotes** [comment]
Not a bad idea, but that requires backend + user login
